dash-renderer/src/persistence.js

```diff
@@ -265,10 +265,22 @@ const noopTransform = {
     apply: (storedValue, _propValue) => storedValue,
 };
 
-const getTransform = (element, propName, propPart) =>
-    propPart
-        ? element.persistenceTransforms[propName][propPart]
-        : noopTransform;
```

Comment:
Old logic checks for just `propPart` for `persistenceTransforms`, but here we add logic to check if either `propName` or `propPart` is the persisted prop.